### PR TITLE
User Info badge and API Added

### PR DESCRIPTION
**Pull request by avdaredevil**
## About
This PR:
- Adds a user badge to the toolbar
- Mocks an API that can provide user image and email

## Preview
image

### Labels
/area front-end
/area centraldashboard
/cc @prodonjs
/assign @avdaredevil

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3297)

### CODE REVIEW

File: components/centraldashboard/public/components/main-page.pug [hunk 1]
```diff
@@ -1,5 +1,5 @@
-iron-ajax(auto, url='/api/platform-info', handle-as='json',
-    on-response='_onPlatformInfoResponse')
+iron-ajax(auto, url='/api/env-info', handle-as='json',
```
Comment (prodonjs) [inline components/centraldashboard/public/components/main-page.pug]: You'll need to update the unit test that mocks this API response.

File: components/centraldashboard/public/components/main-page.js [hunk 2]
```diff
@@ -90,6 +92,11 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
             hideNamespaces: {type: Boolean, value: false, readOnly: true},
             notFoundInIframe: {type: Boolean, value: false, readOnly: true},
             namespace: {type: String, observer: '_namespaceChanged'},
+            __placeholderImage: {
```
Comment (prodonjs) [inline components/centraldashboard/public/components/main-page.js]: It doesn't look like this is being used anymore, right? If so, drop the **/public** segment from the path.

File: components/centraldashboard/app/api.ts [hunk 3]
```diff
@@ -1,26 +1,48 @@
...
+
+interface EnvironmentInfo {
+    user: UserInfo;
+    platform: PlatformInfo;
+}
+
+/** Function that provides user information back */
+function getUserInfo() {
+    return {
+        email: 'blah@blah.com',
```
[the remark this answers is not in the document]
Comment (avdaredevil) [reply components/centraldashboard/app/api.ts]: Makes total sense

File: components/centraldashboard/public/components/main-page.pug (hunk 1 above)
Comment (avdaredevil) [reply components/centraldashboard/public/components/main-page.pug]: Done!

File: components/centraldashboard/public/components/main-page.js (hunk 2 above)
Comment (avdaredevil) [reply components/centraldashboard/public/components/main-page.js]: Just fixed it, and used this value instead.

**Pull request merged**